stylesheets/_component.dropdowns.scss

```diff
@@ -300,3 +300,23 @@
     background-color: darken(color(primary), 15%);
   }
 }
+
+.dropdown--medium ~ ul {
+    width: 300px;
+}
+
+.dropdown__title {
+    border-bottom: 1px solid color(border);
+    margin: 0;
+    padding-bottom: 4px;
+
+    a {
```

Comment:
We can probably achieve this through utility classes rather than overly specific element selectors, barring that, we'd be better some classes specific to the reporting UI